UnitTests/DICOMcloud.Dicom.UnitTest/Helpers/DicomHelpers.cs

```diff
@@ -31,9 +33,14 @@ public DicomHelpers ( )
         
         public static string GetBaseFolder ( ) 
         {
-            string baseFolder = System.AppDomain.CurrentDomain.BaseDirectory ;        
+            string baseFolder = System.AppDomain.CurrentDomain.BaseDirectory ;
+
+
+            var pathRegex = new System.Text.RegularExpressions.Regex(@"\\bin(\\x86|\\x64)?\\(Debug|Release)$", System.Text.RegularExpressions.RegexOptions.Compiled);
```

Comment:
This is not working as expected and the directory returned from this function is now causing all tests to fail. 
What was the problem with the original code?

Reply:
I changed it back for now, the original was giving me one folder above the solution so all the paths didn't work. This was working for me.